app/controllers/users_controller.rb

```diff
@@ -2,10 +2,12 @@
 class UsersController < ApplicationController
   load_and_authorize_resource find_by: :sequence_num, through: :current_company
   def show
+    add_breadcrumb t('users.username', user_name: current_user.name), :user_path
```

Comment:
current_user.name ? Current user can be different from the user on whose show page one is.